Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -1,65 +1,74 @@
-Match checks if the certificates and profiles are valid and will prompt you if there is an issue. By default it is configured to run in readonly mode so it will not regenerate any certificates or profiles if they are not valid.
-
-If you need to build the app locally and run it on a device that is not yet added to the development portal you need to follow these steps:
+Note:- 
+* We have 2 developer teams - Enterprise and App-store 
+* Following options can be used for the command above:
 
-- register the device manually on the portal. Make sure you do that in the correct team: it should be added to the enterprise team, not to the AppStore team
-- run Match to update development certificates and include this new device:
-
-```
-bundle exec fastlane match development --team_id <enterprise team id> --force_for_new_devices --readonly false
-```
+| Code signing identities for | Type param| Team-ID param    |
+|-----------------------------|-----------|-------------------
+|Day-to-day development       |development|Enterprise team ID|
+|Hockeyapp                    |enterprise |Enterprise team ID|
 
-This will regenerate the development profiles and certificates, including all new registered devices in them.
+3. On prompt for password to decrypt the repository, enter the one from 1Password vault.
 
-Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed – as those tasks are usually executed on Circle CI and not locally – but might be necessary for some tasks.
+To troubleshoot, refer to `fastlane/Matchfile`, `fastlane/Appfile` and [documentation](https://docs.fastlane.tools/actions/match/)
 
 ### TestFlight & AppStore
 
 To install certificates and provisioning profiles for creating a Testflight or App Store build locally, run the command
 
 ```shell
- $ bundle exec fastlane match appstore --git_branch <appstore team id> --team_id <appstore team id> --app_identifier <appstore bundle id>
+ $ bundle exec fastlane match appstore --team_id <appstore team id> --git_branch <appstore team id> --app_identifier <appstore bundle id>
 ```
 
-Note: The reason to specify individual app identifiers is that otherwise if AppStore bundle ids are added to the default set of identifiers Match will try to load certificates that may not exist for some apps on the portal, as those apps are registered in different teams.
+Note: 
+- More on `--git-branch` option [below](#several-targets-and-teams)
+- The reason to specify individual app identifiers is that otherwise if AppStore bundle ids are added to the default set of identifiers Match will try to load certificates that may not exist for some apps on the portal, as those apps are registered in different teams.
```

Comment:
```
The reason to specify individual app identifiers is that otherwise if AppStore bundle ids are added to the default set of identifiers Match will try to load certificates that may not exist for some apps on the portal, as those apps are registered in different teams.
```
->
```
We specify individual app identifiers instead of adding all App Store bundle ids to the default set as Match would otherwise attempt to load certificates that do not exist in the portal. This is because some of our app ids are registered with teams that belong to our partners.
```

Reply:
Nice. I was struggling to phrase it nicely myself :)